index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="index.css" />
+</head>
+<body>
+    <main>
+        <header class="header">
+            <div>My Logo</div>
+            <div><img src="Icons/Union.png" alt="burger icon" /></div>
```

Comment:
- [ ] it would be better if you use the `<nav>` tag instead of the `<div>` tag that will give a better sense and also will improve your SEO